### PR TITLE
Update exceptions.txt

### PR DESCRIPTION
**Pull request by mbxIO**
Added 3 news shows to thetvdb exceptions list.

### CODE REVIEW

File: exceptions.txt [hunk 1]
```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers', 'Ballers (2015)', 'Ballers (2014)',
+281708: 'Proof', 'Proof (US)',
+294417: 'Scream The TV Series',
```
Comment (miigotu) [inline exceptions.txt]: Scream The TV Series is already an exception on theTVDB

File: exceptions.txt [hunk 2]
```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers', 'Ballers (2015)', 'Ballers (2014)',
```
Comment (miigotu) [inline exceptions.txt]: Should be:
281714: 'Ballers (2015)', 'Ballers (2014)'

'Ballers' is already the name of the show on theTVDB, so adding an exception to that makes SR search for the same  name 2x.

File: exceptions.txt (hunk 1 above)
Comment (mbxIO) [inline exceptions.txt]: @miigotu I looked at the list and I don't see a scene exception for Scream... I'm super new to this, so I'm sure I'm missing it. 

btw, trying to figure this out on my own. I'd love to contribute to the scene exceptions list and/or other areas due to the ridiculous amount of TV series I download/watch ... I'm just a little new to github and SR. 

(ps - just sent a donation. But, it is coming from a different email. Thanks again)

File: exceptions.txt (hunk 2 above)
Comment (mbxIO) [inline exceptions.txt]: I removed Ballers so it doesn't search 2x ... Now, how would this list go about being merged with the "main one".

File: exceptions.txt [hunk 3]
```diff
@@ -419,3 +419,6 @@
 294973: 'Home Fires UK',
 295438: 'Shark UK',
 148581: 'Strike Back Legacy',
+281714: 'Ballers (2015)', 'Ballers (2014)',
+281708: 'Proof', 'Proof (US)',
+294417: 'Scream The TV Series',
```
Comment (miigotu) [inline exceptions.txt]: image
The scream exception is not needed, thetvdb already has this exception, so adding it here is a double excpetion.

**Comment by miigotu** [comment]
Main one? You mean directly to theTVDB? We have these lists because theTVDB doesnt let people edit things or get things added.

**Comment by mbxIO** [comment]
@miigotu No, this exception.txt list. I thought this list I was adding to would eventually be merged with the "main" exception.txt which I'm assuming gets downloaded to SR somehow? But, I am new to SR and know very little.

**Comment by miigotu** [comment]
Ah, it will get updated in SR after every change, on the next commit.

**Comment by miigotu** [comment]
I think I am going to change the way I am handling that soonish, and have it go back to downloading it.

**Comment by mbxIO** [comment]
@miigotu So, when I add shows to the list, how do I "submit" that list to you so that it stays up-to-date? This pull request way that I've been doing (hopefully correctly)? 

I don't know how often it's updated or if you are the one who normally handles it but I am obsessed with keeping Plex up-to-date with the most recent and newest TV shows for myself, the girlfriend and the 10+ people I share my Plex server with. 291 TV series and counting. I understand, for the most, the lingo of how to edit the list ... I just need a little assistance on navigating on github.

**Comment by miigotu** [comment]
This is the correct way, just make sure the exception is not already on theTVDB. If you see in my screenshot there it shows exceptions under the regular title on theTVDB.
image

**Comment by mbxIO** [comment]
@miigotu Yea, I understand now. I'll check between TVDB and a few torrent / nzb sites before submitting my updated exceptions list from here on in. 

I wanted to send you at least $20 for the help but the bank account was a little lower than I excepted :( lol so you will be receiving another donation soon. But, I did want to ask you about something, well 2 things really, but one will answer itself. I posted #2122 in issues, which is closed now, and I closed because of the comment that was left. I didn't was to cause any issues because I need and will take any help that I can get with SR. But, 1. I can guess why the link was left, basically saying don't post basic dumb shit in here, what exactly about that issue I posted is the problem and 2. Can you just tell me WTF is causing that ... I continuously get that because, I'm assuming, when I download torrents, it downloads outside of my "tv folder" and I manually have to drag the tv show into the "tv folder". I only get this error with when SR picks torrents and it only happens with some files and not others ... but it's happening more now then ever.

**Comment by miigotu** [comment]
`I can guess why the link was left` Because we have guidelines for what information we need provided when you ask a question or have a problem/bug/issue. He just meant to go read that and post the information you were missing, because without it it is long and drawn out string of questions and day later answers, which makes it almost impossible to help you.

torrent client should be downloading in 1 folder, lets call it `Downloads/`
client should move completed downloads to a different folder. lets call it `Downloads/Complete`
SR should be post processing the `Downloads/Complete` folder, and in your post processor settings, you set whether you want to move/copy/hard link/or soft link - and also other settings like delete folders and extra files.

If you use copy, it will not delete the file or folder.

The best way, if your TVShows library is on the same partition as your `Downloads/Complete`, you should probably use the hard link method, so you can keep seeding. This does not use any extra space but you have what looks like 2 files, one in `Downloads/Complete` and one in your TVShows library that are actually the same file. If you later delete the reference in `Downloads/Complete`, the one in TVShows remains.

**Comment by miigotu** [comment]
BTW, donations are not required around here to get help, but if you do end up donating, thanks in advance.

**Comment by mbxIO** [comment]
@miigotu I am currently running 10TB external HDs that are separate so the Copy method may work but I heard SR isn't very good at renaming with this method. Is this true?

Last question about the exceptions list... When I submit a request to you, does that updated list show as the new list on SiCKRAGETV/sb_tvdb_scene_exceptions page? and when someone sends a request, how long does it usually take for that request to be approved? (if it even works that way)

**Comment by miigotu** [comment]
Untrue, SR works the same with renaming regardless of processing method.

If you are using Copy method, the files remain at the source. If you don't want two copies, you need to use the Move method.

`When I submit a request to you, does that updated list show as the new list on SiCKRAGETV/sb_tvdb_scene_exceptions page?`

As soon as i merge it yes. The pull request is usually merged in a few hours if everything is proper in the pull request without errors as we have to make sure to not break the list.

**Comment by mbxIO** [comment]
@miigotu I think I F'd up when trying to submit a new update with scream removed and broke ass added so I think i got it right this time. Just wanted to let you know I updated the list for the final time for now. 

So, is there a SR discussion board or anything on here? How do you go about having a convo since there really isn't a messaging system? I tried to push a feature request for "Preferred Words" Search on Archive and that died as fast as the convo did lol.

**Comment by mbxIO** [comment]
@miigotu Hey quick question about the updated exceptions list. I know that our lists were merged and I am now updated to the newest commit but I am still on the old exceptions list. Any idea why it did not download with the new update? 

capture

**Comment by miigotu** [comment]
@mbxplex I have to manually copy the exceptions to the other repo and commit it for it to be pushed out to users.

I have not done this yet because tonight I am changing how it works completely, so that I no longer need to do it manually every PR.

**Comment by mbxIO** [comment]
@miigotu lol completely makes sense. I just wanted to make sure I didn't miss something. Thanks!

**Pull request merged**